### ISSUE TITLE
FIXED: Node 11 Error: ReferenceError: internalBinding is not defined

### ISSUE THREAD

**Issue by e2tha-e** [electric-eloquence/fepper#15]
As of Node 11.0.0, users will get this error running`require('gulp')` (3.9.1). Fepper requires gulp for running tasks, and for providing the architecture for extensions.

```
fs.js:25
'use strict';
^

ReferenceError: internalBinding is not defined
    at fs.js:25:1
    at req_ (/Users/ee/Sites/Vagrant/Trusty64/VHosts/electric-eloquence/Other/Fepper-install-test/fepper-base/node_modules/natives/index.js:137:5)
    at Object.req [as require] (/Users/ee/Sites/Vagrant/Trusty64/VHosts/electric-eloquence/Other/Fepper-install-test/fepper-base/node_modules/natives/index.js:54:10)
    at Object.<anonymous> (/Users/ee/Sites/Vagrant/Trusty64/VHosts/electric-eloquence/Other/Fepper-install-test/fepper-base/node_modules/vinyl-fs/node_modules/graceful-fs/fs.js:1:99)
    at Module._compile (internal/modules/cjs/loader.js:707:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:718:10)
    at Module.load (internal/modules/cjs/loader.js:605:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:544:12)
    at Function.Module._load (internal/modules/cjs/loader.js:536:3)
    at Module.require (internal/modules/cjs/loader.js:643:17)
```
Fepper has no immediate plans to upgrade gulp beyond 3.x.x. The existing gulp plugin ecosystem was not developed with 4.x.x forward-compatibility in mind. Whether the ecosystem moves itself along to 4.x.x-compatibility is uncertain.

Nonetheless, a solution is being worked out and we will do everything possible to make Fepper compatible with the latest Node version. In the meantime, those on Node 11.x.x will need to downgrade their Node version.

**Comment by e2tha-e** [comment]
Related to https://github.com/gulpjs/gulp/issues/2246

**Comment by e2tha-e** [comment]
This has been fixed as of v0.7.0 on all Fepper distributions. It was fixed by updating a nested dependency of gulp, [natives](https://www.npmjs.com/package/natives) to v1.1.6.

natives@1.1.6 was released on 2018-09-12. The Fepper `package-lock.json` didn't pick up this release until 2018-10-31.

Fepper looks all clear for Node v11.

**Comment by e2tha-e** [comment]
[Why you may want to remain on gulp 3 and how to do so securely.](https://github.com/electric-eloquence/gulp/blob/v3-lts/docs/why-gulp-3.md)
